### ISSUE TITLE
buildah build-using-dockerfile not work

### ISSUE THREAD

**Issue by gouyang** [projectatomic/buildah#78]
```
$ sudo ./buildah bud .
STEP 1: FROM fedora
STEP 2: RUN dnf -y update && dnf clean all
json: cannot unmarshal array into Go struct field Process.capabilities of type specs.LinuxCapabilities
ERRO[0000] error building: error building at step {Env:[PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin DISTTAG=f25docker FGC=f25 FBR=f25] Command:run Args:[dnf -y update && dnf clean all] Flags:[] Attrs:map[] Message:RUN dnf -y update && dnf clean all Original:RUN dnf -y update && dnf clean all}: exit status 1 

$ cat Dockerfile 
FROM fedora

RUN dnf -y update && dnf clean all
RUN dnf -y install nginx && RUN dnf clean all
RUN echo "daemon off" >> /etc/nginx/nginx.conf
RUN echo "nginx on Fedora" > /usr/share/nginx/html/index.html

EXPOSE 80

CMD [ "/usr/sbin/nginx" ]
```

**Comment by nalind** [comment]
Yup, that's a known incompatibility with the latest version of runc (more specifically, the version of the runtime spec that it conforms to).  Now that we're not vendoring as much of docker, we should be able to pull in a newer version of the spec and the library that we use to generate info for it.  I'll note this issue in the PR for doing that, when we open it.

**Comment by nalind** [comment]
Should be fixed by #79.

**Comment by nalind** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [(8) [Buildah] Add "dockah build-using-dockerfile" aka "buildah bud"](https://trello.com/c/AyBXqoRY/443-8-buildah-add-dockah-build-using-dockerfile-aka-buildah-bud)
